dockstore-webservice/src/main/java/io/dockstore/webservice/core/metrics/ExecutionTimeStatisticMetric.java

```diff
@@ -34,21 +38,72 @@
 public class ExecutionTimeStatisticMetric extends StatisticMetric {
     public static final String UNIT = "s"; // Store in seconds
 
+
+    //TODO: could consider moving these down the hierarchy for cost, cpu, memory if those become well-populated
+    //TODO: could also consider storing these as an array/record
+    @Column(nullable = false)
+    @NotNull
+    @ApiModelProperty(value = "The 50th percentile value from the data points", required = true)
```

Comment:
We can probably ditch the `ApiModelProperty` annotation (here and below) if it's used exclusively to generate the olde Swagger yaml (I think it is).